src/contour.rs

```diff
@@ -201,27 +198,84 @@ impl ContourBuilder {
                 for polygon in &mut polygons {
-                    if contains(&polygon[0], &hole) != -1 {
-                        polygon.push(hole);
+                    if contains(&polygon.exterior().0, &hole.0) != -1 {
+                        polygon.interiors_push(hole);
                         return;
                     }
                 }
             })
             .for_each(drop);
 
-        let mut properties = Map::with_capacity(1);
-        properties.insert(String::from("value"), to_value(threshold)?);
-        Ok(Feature {
-            geometry: Some(Geometry {
-                value: MultiPolygon(polygons),
-                bbox: None,
-                foreign_members: None,
-            }),
-            properties: Some(properties),
+        Ok(Contour {
+            geometry: MultiPolygon(polygons),
+            threshold,
+        })
+    }
+}
+
+/// A contour has the geometry and threshold of a contour ring, built by [`ContourBuilder`].
+#[derive(Debug, Clone)]
+pub struct Contour {
+    geometry: MultiPolygon,
+    threshold: f64,
+}
+
+impl Contour {
+    /// Borrow the [`MultiPolygon`](geo_types::MultiPolygon) geometry of this contour.
+    pub fn geometry(&self) -> &MultiPolygon {
+        &self.geometry
+    }
+
+    /// Get the owned polygons and threshold of this countour.
+    pub fn into_inner(self) -> (MultiPolygon, f64) {
+        (self.geometry, self.threshold)
+    }
+
+    /// Get the threshold used to construct this contour.
+    pub fn threshold(&self) -> f64 {
+        self.threshold
+    }
+
+    #[cfg(feature = "geojson")]
+    /// Convert the contour to a struct from the `geojson` crate.
+    ///
+    /// To get a string reresentation, call to_geojson().to_string().
+    /// ```
+    /// use contour::ContourBuilder;
+    ///
+    /// let builder = ContourBuilder::new(10, 10, false);
+    /// # #[rustfmt::skip]
+    /// let contours = builder.contours(&[
+    /// // ...ellided for brevity
+    /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 2., 1., 2., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 2., 2., 2., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 1., 2., 1., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 2., 2., 2., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 2., 1., 2., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+    /// #     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.
+    /// ], &[0.5]).unwrap();
+    ///
+    /// let geojson_string = contours[0].to_geojson().to_string();
+    ///
+    /// assert_eq!(&geojson_string[0..27], r#"{"geometry":{"coordinates":"#);
+    /// ```
+    pub fn to_geojson(&self) -> geojson::Feature {
+        let mut properties = geojson::JsonObject::with_capacity(1);
+        // REVIEW: This maintains existing behavior, but should we rename
+        // `value` to something more explicable, like `threshold`?
```

Comment:
Yes I think it would be good to change the name of the field from "value" to "threshold", it would be more explicit and it would correspond better to our new internal representation whose field is now called "threshold".